utah-design-system-website/src/react/components/websiteContent/library/components/buttons/button/ButtonDocumentation.jsx

```diff
@@ -64,27 +64,27 @@ function ButtonDocumentation() {
             <Button
               appearance="solid"
               color="primary"
-              onClick={useCallback(() => showBanner({ message: 'primary emphasized button clicked' }), [showBanner])}
+              onClick={useCallback(() => addBanner({ message: 'Primary emphasized button clicked', duration: 3500 }), [addBanner])}
```

Comment:
Make duration global.
Make durations global and modifiable by the programmer.

On the create banner context pass in the value for the banner duration.